.devcontainer/codespaces/devcontainer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "nextflow-training (codespaces)",
+
+    // Update this to pin the release tag before release
+    "image": "ghcr.io/nextflow-io/training:latest",
+    // Uncomment this and comment "image" when testing Dockerfile changes
+    // "build": { "dockerfile": "../Dockerfile", "context": ".." },
+
+    "features": {
+        "ghcr.io/devcontainers/features/docker-outside-of-docker:1": {
+            "version": "latest"
+        }
+    },
+    "workspaceFolder": "/workspaces/training",
+    "remoteEnv": {
+        "NXF_HOME": "/workspaces/.nextflow",
+        "HOST_PROJECT_PATH": "/workspaces/training"
+    },
+    "customizations": {
+        "vscode": {
+            "settings": {
+                "python.defaultInterpreterPath": "/opt/conda/bin/python"
+            },
+            "extensions": [
+                "ms-python.python",
+                "ms-python.vscode-pylance",
+                "nf-core.nf-core-extensionpack", // nf-core recommended extensions
+                "nextflow.nextflow", // Nextflow VS Code extension
+                "codezombiech.gitignore", // Language support for .gitignore files
+                "cssho.vscode-svgviewer" // SVG viewer
```

Comment:
I think we can simplify these - `nextflow.nextflow` shouldn't be needed as it's already part of the nf-core extension pack, and the other two could be added to that. Then we just have the nf-core extension pack and be done with it.

I'd also like to remove a couple of the more subjective packages from the nf-core extension pack (rainbow indentation, TODO tree, code spell checker). I find myself disabling these each time in devcontainers, which is annoying.

We can do this in a separate PR, as needs upstream changes to the nf-core extension pack first.

Reply:
https://github.com/nf-core/vscode-extensionpack/pull/25